README.md

```diff
@@ -58,7 +58,7 @@ Use the `artifact_sources` input variable to limit the downloads to a set of sta
 
 | Environment Variable | Description |
 | --- | --- |
-| `PULLED_ARTIFACT_LOCATIONS` | An absolute path list of the downloaded artifacts. The list is separated with newlines (\n) |
+| `ARTIFACT_PATHS` | An absolute path list of the downloaded artifacts. The list is separated with newlines (\n) |
```

Comment:
Shouldn't we prefix this with `BITRISE` same way we do with input variables?

Reply:
I'm not sure about naming conventions but based on other step outputs we should (e.g. https://github.com/bitrise-steplib/steps-xcode-archive/blob/eb2baab896e6549cad464c27ebc43a4aaa666ed1/step.yml#L378 - all prefixed with `BITRISE`)